x/committee/keeper/proposal_test.go

```diff
@@ -818,8 +819,28 @@ func (suite *KeeperTestSuite) TestCloseProposal() {
 	// Confirm proposal exists
 	proposal, found := keeper.GetProposal(ctx, proposalID)
 	suite.True(found)
+
 	// Close proposal
 	keeper.CloseProposal(ctx, proposal, types.Passed)
+
+	events := ctx.EventManager().Events()
+	event := events[0]
+	suite.Require().Equal("proposal_close", event.Type)
+
+	hasProposalTallyAttr := false
+	for _, attr := range event.Attributes {
+		if string(attr.GetKey()) == "proposal_tally" {
+			hasProposalTallyAttr = true
+			valueStr := string(attr.GetValue())
+			suite.Require().True(strings.Contains(valueStr, "Yes votes:"))
+			suite.Require().True(strings.Contains(valueStr, "Current votes:"))
+			suite.Require().True(strings.Contains(valueStr, "Possible votes:"))
+			suite.Require().True(strings.Contains(valueStr, "Vote threshold:"))
+			suite.Require().True(strings.Contains(valueStr, "Quorum:"))
```

Comment:
hot tip: there's a `suite.Contains` method that will check if a string contains another string. It'll have nicer error messages and saves some typing.
https://pkg.go.dev/github.com/stretchr/testify@v1.7.0/assert#Contains

Reply:
Updated in https://github.com/Kava-Labs/kava/pull/919/commits/afa5818c1a7ef55529d892006f8d2e7bf1e1c648